src/EditorFeatures/CSharpTest/IntroduceUsingStatement/IntroduceUsingStatementTests.cs

```diff
@@ -23,7 +23,6 @@ protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspa
         [InlineData("var name[||] = disposable;")]
         [InlineData("var name [||]= disposable;")]
         [InlineData("var name =[||] disposable;")]
-        [InlineData("var name = [||]disposable;")]
```

Comment:
Interesting.. I would expect putting my cursor just before the disposable creation also offers me to wrap the declaration inside a using statement. Can we special case this to avoid a breaking change here?

Reply:
As said in `Question` section, I'm unsure about this one (wanted to commit it to PR to gather feedback). If the concecsus is that we want to enable it here, we surely can.

Reply:
Tagging @jnm2 who contributed this refactoring and @CyrusNajmabadi who reviewed it.
I like the original behavior that we offer the refactoring on disposable creation initializer.

Reply:
Keeping it would be my preference. Otherwise there's a strange hole. I can't say for sure where I place my cursor when I use the refactoring.

Reply:
Sure, will update accordingly.